third_party/upb/upb/port/def.inc

```diff
@@ -327,3 +327,30 @@ void __asan_unpoison_memory_region(void const volatile *addr, size_t size);
 #else
 #define UPB_DESC(sym) google_protobuf_##sym
 #endif
+
```

Comment:
All files under `third_party/upb' and `src/core/ext/upb-generated' are not allowed to be modified as they belong to other project. Would you undo this change?

Reply:
I can do this for this PR, but then the shared build is not going to work anymore. So I guess we need to raise this with protobuf?

Reply:
For `third_party/upb/*`, https://github.com/protocolbuffers/upb is the right place to make this change. We're going to use git-submodule instead of git-subtree for upb soon. Then all changes made here will be discarded.

For `src/core/ext/upb-generated/*`, why do we need to export those symbols? Those upb generated files are supposed to be private so I don't understand why this is required. If there is some case where those symbol is part of gRPC public API, we can change it not to do that.

Reply:
> For `src/core/ext/upb-generated/*`, why do we need to export those symbols? Those upb generated files are supposed to be private so I don't understand why this is required. If there is some case where those symbol is part of gRPC public API, we can change it not to do that.

This must be used somewhere; I only added the minimal things to avoid build failure.